frontend/e2e-test/test/e2e/specs/0_citizen/citizen-decisions.spec.ts

```diff
@@ -0,0 +1,74 @@
+// SPDX-FileCopyrightText: 2017-2020 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import { logConsoleMessages } from '../../utils/fixture'
+import { enduserRole } from '../../config/users'
+import CitizenHomePage from '../../pages/citizen/citizen-homepage'
+import { initializeAreaAndPersonData } from '../../dev-api/data-init'
+import { deleteApplication, insertApplications } from '../../dev-api'
+import {
+  applicationFixture,
+  enduserChildFixtureJari,
+  enduserGuardianFixture,
+  Fixture,
+  EmployeeBuilder
+} from '../../dev-api/fixtures'
+import CitizenDecisionsPage from '../../pages/citizen/citizen-decisions'
+import { format } from 'date-fns'
+
+const citizenHomePage = new CitizenHomePage()
+const citizenDecisionsPage = new CitizenDecisionsPage()
+
+let applicationId: string
+let employee: EmployeeBuilder
+
+fixture('Citizen decisions')
+  .meta({ type: 'regression', subType: 'citizen-decisions' })
+  .before(async () => {
+    await initializeAreaAndPersonData()
```

Comment:
This data is not cleaned up afterwards. The function returns a function that does it.

Reply:
Yes - which is "await Fixture.cleanup()" which is called in the suite cleanup. But agree that it is not obvious, so will add a redundant call via the function returned by initializeAreaAndPersonData 👍

Reply:
Oh, nevermind then. You don't need to add the function call, I just didn't know `Fixture.cleanup` does that. There's something else wrong with the clean up then. It definitely has worked for other test fixtures before.